.github/workflows/docker.yml

```diff
@@ -74,14 +72,26 @@ jobs:
             echo "PECAN_VERSION=develop" >> $GITHUB_ENV
             echo "PECAN_TAGS=develop" >> $GITHUB_ENV
           fi
-
+      # If any dependencies changed in this PR, try to update depends image
+      # Only runs on pull_request -- others it's not clear what base to compare to
+      - name: check for changed dependencies
+        if: github.base_ref != ''
+        run: |
+          dep_chgs=$(
+            git diff --name-status ${GITHUB_BASE_REF}...${GITHUB_HEAD_REF} -- docker/depends)
```

Comment:
Currently getting "fatal: bad revision develop...allow-depends-update" in the GitHub run but not locally... investigating.